tests/Doctrine/Tests/Common/Collections/AbstractLazyCollectionTest.php

```diff
@@ -35,7 +34,7 @@ private function buildCollection(array $elements): LazyArrayCollection
     public function testClearInitializes(): void
     {
         $collection = $this->buildCollection(['a', 'b', 'c']);
-        assert($collection instanceof LazyArrayCollection);
+        self::assertInstanceOf(LazyArrayCollection::class, $collection);
```

Comment:
I believe, these changes should be reverted. Otherwise we would test the test.

These `assert()` calls are merely a help for the static code analysis.

Reply:
Ok, tks.